### ISSUE TITLE
Search UI consistency

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#21055]
## User Story

_As a user I would like for all the various search tools on VA.gov to have a consistent look and feel, consistent behavior and return results in a consistent way so that it is not necessary to form a new mental model for each new tool.

## Expected Behavior

The global VA.gov, Research and Support, Find a VA Form, Find a Yellow Ribbon, and other "micro" searches within VA.gov look and function in a consistent manner.

## Acceptance Criteria

- [ ] Options are mocked up
- [ ] Options are reviewed by VSA and VSP accessibility
- [ ] Options are reviewed by design lead

**Comment by MarciMcGuireGCIO** [comment]
@denisecoveyduc do you mind tagging Megan Gayle onto this GH issue?  I don't know her GH handle. Thx!

**Comment by denisecoveyduc** [comment]
@MarciMcGuireGCIO absolutely.. it is @mgleo  ;)

**Comment by mgleo** [comment]
Here is a list of search tools that we know about. Some of them are still using the old layout. 

On [main search sidebar](https://www.va.gov/search/?query=health):
[Look up Board of Veterans' Appeals (BVA) decisions](https://www.index.va.gov/search/va/bva.jsp)
[Search VA reference materials (WARMS)](https://www.index.va.gov/search/va/va_adv_search.jsp?SQ=www.benefits.va.gov/warms)
[Find a VA form](https://www.va.gov/find-forms/)
[VA handbooks and other publications](https://www.va.gov/vapubs/)
[Explore and apply for open VA jobs](https://www.vacareers.va.gov/job-search/index.asp)

More search pages that we know about:
[GI Bill® Comparison Tool](https://www.va.gov/gi-bill-comparison-tool)
[Find a Yellow Ribbon school](https://www.va.gov/education/yellow-ribbon-participating-schools/)
[Resources and Support](https://www.va.gov/resources/)
[Grave Locator](https://m.va.gov/gravelocator/index.cfm)
[Find Locations](https://www.va.gov/directory/guide/home.asp)
[Find a Facility](https://www.va.gov/find-locations)

**Comment by MarciMcGuireGCIO** [comment]
@rtwell tagging you in so you can see the above list of search tools that Megan has discovered thus far.

**Comment by MarciMcGuireGCIO** [comment]
@callen2563 when you get a chance, can you please add the links to your new mockups to this ticket?  Thanks!

**Comment by callen2563** [comment]
FAF Update - I have just updated the InVision mocks to have the 17pt Serif, Bitter font that the unauthenticated team and I decided on based on the current styling on the majority of header stylings. I'm working with the design system team to update the action link to a square font awesome icon so that it is in alignment with the link next to it. 

[Find a Form mobile ](https://vsateams.invisionapp.com/share/8Y10IE2JN2J4#/447613733_Find_A_Form_-_Mobile_3)

[Find a Form desktop](https://vsateams.invisionapp.com/share/SD10IE31CENR#/447613826_Find_A_Form_-_Desktop_3)

**Comment by MarciMcGuireGCIO** [comment]
@callen2563 @mgleo - we will create separate issues for remaining work, so you can add your comments / mockup links to those vs. putting them here in the Epic.  Thx!!

**Comment by mgleo** [comment]
Regarding the 508 issue of adding underlines to header links: We were following the header link style defined at the bottom of the benefits pages [seen here](https://www.va.gov/disability/eligibility/). If we change this style it will introduce another style inconsistency on VA.gov. For our use case, users are searching for keywords or phrases and would expect a list of links after searching. Thoughts on this? @joshkimux @sshein

**Comment by joshkimux** [comment]
@mgleo Hi Megan! In terms of consistency, [sitewide we're moving towards having underlines under headings that are links per ticket 3556](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556#issuecomment-756432528).

**Comment by mgleo** [comment]
Got it, thanks @joshkimux!

**Comment by callen2563** [comment]
@CrystabelReiter I'm wondering what the rule is around download links do they need to have the download icon? Also, wouldn't a download be a secondary button?
<img width="864" alt="Screen Shot 2021-04-09 at 1 22 46 PM" src="https://user-images.githubusercontent.com/68560244/114218713-c60e2980-9937-11eb-9e79-54ecf33218fc.png">

**Comment by CrystabelReiter** [comment]
@callen2563 Yes, I'd include a download icon and downloading things like PDFs are still links because we're pointing to a resource with a unique URL that we're offloading instead of opening in a browser. Thanks for asking!

**Comment by callen2563** [comment]
Ok, got it thanks!

**Comment by MarciMcGuireGCIO** [comment]
Noting that the GH issue for documenting the design pattern is here:  department-of-veterans-affairs/vets-design-system-documentation#459

**Comment by MarciMcGuireGCIO** [comment]
Experimental Design System Request: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/503

**Comment by jilladams** [comment]
Because Zenhub is likely going away, Sitewide may hit our Github Projects issue cap (1200), and may not be able to port all our issues in right away, logging which issues in this ZH epic are still open, so we can later build parent/child relationships in GHP as needed: 
* #30399
* #24597
